core/src/main/java/cz/xtf/core/waiting/helpers/FailFastBoolSupplierBuilder.java

```diff
@@ -0,0 +1,149 @@
+package cz.xtf.core.waiting.helpers;
+
+
+import cz.xtf.core.config.BuildManagerConfig;
+import cz.xtf.core.config.OpenShiftConfig;
+import cz.xtf.core.event.EventList;
+import cz.xtf.core.event.EventListFilter;
+import cz.xtf.core.openshift.OpenShift;
+import cz.xtf.core.openshift.OpenShifts;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BooleanSupplier;
+
+public class FailFastBoolSupplierBuilder {
+
+	private final List<OpenShift> openShifts = new ArrayList<>();
+	private final List<BooleanSupplier> suppliers = new ArrayList<>();
+	private long intervalMilis;
+
+	private FailFastBoolSupplierBuilder(String... namespaces) {
+		intervalMilis = 1_000L;
+		if (namespaces.length == 0) {
+			openShifts.add(OpenShifts.master());
+		} else {
+			for (String namespace : namespaces) {
+				openShifts.add(OpenShifts.master(namespace));
+			}
+		}
+	}
+
+	public static FailFastBoolSupplierBuilder ofOpenshiftAndBmNamespace() {
+		return ofNamespaces(OpenShiftConfig.namespace(), BuildManagerConfig.namespace());
+	}
+
+	public static FailFastBoolSupplierBuilder ofNamespaces(String... namespaces) {
+		return new FailFastBoolSupplierBuilder(namespaces);
+	}
+
+	public EventFailFastBoolSupplierBuilder events() {
+		return new EventFailFastBoolSupplierBuilder(this);
+	}
+
+	private void addBoolSupplier(BooleanSupplier supplier) {
+		this.suppliers.add(supplier);
+	}
+
+	public FailFastBoolSupplierBuilder interval(long intervalMilis) {
```

Comment:
1, `intervalMilis` should be `intervalMillis`.

Reply:
fixed